EventFilter/CSCTFRawToDigi/src/CSCTFEvent.cc

```diff
@@ -3,7 +3,7 @@
 #include <string.h>
 #include <stdexcept>
 
-unsigned int CSCTFEvent::unpack(const unsigned short *buf, unsigned int length) throw() {
+unsigned int CSCTFEvent::unpack(const unsigned short *buf, unsigned int length) noexcept(false) {
```

Comment:
I agree, I've used the noexcept for destructors before that, seems I used it here 'by habit' instead of just remove the throw specifier. But since I think to overwrite the old commit, would you like me to wait for Chris to respond first or not ?